www/source/index.html.slim

```diff
@@ -27,24 +27,22 @@ header.bg-gradient.margin-top-offset.short-bg.relative
   /! announcement
   .row.columns
     .box-white.shadow.z-20.slide-left
-      h3.large-11.medium-11 Announcing InSpec 3.0
-      p.large-11.medium-11 Plugin system, global attributes, enchanced skip messaging, and more.
+      h3.large-11.medium-11 Announcing InSpec 4
+      p.large-11.medium-11 Numerous bugfixes and new features.
```

Comment:
"Inputs system overhaul, updated AWS-SDK to v3, support for the new Chef License, and many bugfixes and minor features"